.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 language: java
 script: mvn verify
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
Changing from `oraclejdk8` to `openjdk8`, because `oraclejdk8` is not installed on `Xenial`, which is the default distribution chosen by travis nowadays... F.Y.I @nzroller.